documentation/Ingress-controller-upgrade.md

```diff
@@ -0,0 +1,36 @@
+# Ingress Controller Upgrade
+
+**Pre-requisits**
+- AKS Cluster with the ingress controller you wish to upgrade
+- An app deployed to the cluster to monitor i.e. apply, register etc
+- Set you AKS context for the cluster `az aks get-credentials ...`
```

Comment:
Use the `make ... get-cluster-credentials` instead (link to https://github.com/DFE-Digital/teacher-services-cloud#kubectl)